### PR TITLE
rna-transcription: abbreviate to 'rna' and 'dna'

### PR DESCRIPTION
**Pull request by shingtaklam1324**
Also ran the example code and tests through `rustfmt`

**Reference Issues**
#407

### CODE REVIEW

File: exercises/rna-transcription/example.rs [hunk 1]
```diff
@@ -20,19 +22,26 @@ fn transcribe_dna_rna(c: char) -> Option<char> {
...
-        DeoxyribonucleicAcid { nucleotides: nucleotides.to_string() }
+impl DNA {
+    pub fn new(nucleotides: &str) -> DNA {
+        DNA {
+            nucleotides: nucleotides.to_string(),
+        }
     }
 
-    pub fn to_rna(&self) -> Result<RibonucleicAcid, ()> {
-        let rna_nucleotides: String = self.nucleotides.chars().filter_map(transcribe_dna_rna).collect();
```
Comment (petertseng) [inline exercises/rna-transcription/example.rs]: unrelated line change caused by fmt.

File: exercises/rna-transcription/example.rs [hunk 2]
```diff
@@ -20,19 +22,26 @@ fn transcribe_dna_rna(c: char) -> Option<char> {
         'G' => Some('C'),
         'A' => Some('U'),
         'T' => Some('A'),
-        _   => None
+        _ => None,
```
Comment (petertseng) [inline exercises/rna-transcription/example.rs]: unrelated line change caused by fmt.

File: exercises/rna-transcription/example.rs [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+            nucleotides: nucleotides.to_string(),
+        }
     }
 }
 
 #[derive(PartialEq, Eq, Debug)]
-pub struct DeoxyribonucleicAcid {
-    nucleotides: String
+pub struct DNA {
+    nucleotides: String,
```
Comment (petertseng) [inline exercises/rna-transcription/example.rs]: unrelated line change caused by fmt.

File: exercises/rna-transcription/example.rs [hunk 4]
```diff
@@ -1,17 +1,19 @@
 #[derive(PartialEq, Eq, Debug)]
-pub struct RibonucleicAcid {
-    nucleotides: String
+pub struct RNA {
+    nucleotides: String,
```
Comment (petertseng) [inline exercises/rna-transcription/example.rs]: unrelated line change caused by fmt.

**Comment by shingtaklam1324** [comment]
Un-`rustfmt`-ed all of the code.

**Comment by shingtaklam1324** [comment]
Un-`rustfmt`-ed all of the code.

**Comment by petertseng** [comment]
Thank you. Feel free to submit a PR for rustfmt changes if you like. One commit that `rustfmt` all exercises is fine, I think. Note some specific lines we did not want to format in https://github.com/exercism/rust/pull/143 though.

**Comment by shingtaklam1324** [comment]
Thanks, given that none of the rest of the repository uses `rustfmt`, it makes sense to keep it so for consistency reasons. If I have some free time, I may write a bash script to handle `cargo fmt`-ing all of the code.

**Comment by shingtaklam1324** [comment]
For #143 , `rustfmt` can have use the crate attribute `#[cfg_attr(rustfmt, rustfmt_skip)]` to skip the file. It just needs to be added to the crate root.
